### PR TITLE
Remove some inline HTML from MD docs

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

I removed [some inline HTML](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md033---inline-html) that had corresponding MD equivalents.
In most cases, replacing `<br/>` with double spaces at the end of a line.

### CODE REVIEW

File: Documentation/Input/Pointers.md [hunk 1]
```diff
@@ -72,7 +72,6 @@ The *TouchPointer* is responsible for working with Unity Touch input (i.e. touch
 The *MousePointer* powers a screen to world raycast for far interactions, but for mouse instead of touch.
 
 <img src="../../Documentation/Images/Pointers/MRTK_MousePointer.png" width="400">
-<br/>
```
Comment (Troy-Ferrell) [inline Documentation/Input/Pointers.md]: I think without this it actually looks weird

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: Do you have a screenshot? I don't see much of a difference on my end

Comment (Troy-Ferrell) [reply Documentation/Input/Pointers.md]: I recall seeing after publishing once I could view it on https://microsoft.github.io/

I'm not sure how you can view a private branch on https://microsoft.github.io/?

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: What I see, in the MS Docs renderer for VS Code (which I think should mostly match our github.io styling?):

As-is:
image

With this change:
image

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: >I recall seeing after publishing once I could view it

Ahh okay. I'll build DocFx locally and check

Comment (Troy-Ferrell) [reply Documentation/Input/Pointers.md]: Yea I've learned not to trust vs code unfortunately

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: Do you have https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-preview?

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: Ah you're right it looks different in the generated doc (more compressed), but I feel like it looks the same on https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/Pointers.html#far-pointers?

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: image

Comment (Troy-Ferrell) [reply Documentation/Input/Pointers.md]: hmm try switching html img tag to markdown

Comment (keveleigh) [reply Documentation/Input/Pointers.md]: I replaced the img tag with markdown and resized the image to the 400 pixels specified in the img tag

**Pull request merged**